developers/pymavlink.md

```diff
@@ -12,12 +12,6 @@ All system components that communicate via MAVLink are expected to send a HEARTB
 
 When the autopilot is being commanded to move via RC_CHANNELS_RAW or MANUAL_CONTROL messages, the messages must be sent at a constant rate like the HEARTBEAT message. Otherwise, the autopilot will execute a failsafe if it has not received an updated command after a timeout period.
 
-## Recommendation
```

Comment:
Maybe we should add a note saying that python2 is not recommended.

Reply:
I think both should work fine for now. When issues start arising because of Python2, we can add the the note.